hooks/useTokenVotes.ts

```diff
@@ -9,21 +8,21 @@ export const useTokenVotes = (address?: Address) => {
     contracts: [
       {
         chainId: PUB_CHAIN.id,
-        abi: iVotesAbi,
+        abi: {} as any,
```

Comment:
Using `any` is a bad practice.

Reply:
I deleted this hook because it is not used in maciVoting